src/ThirdParty/Gitlab/Adapter/GitLabIssueTracker.php

```diff
@@ -190,7 +196,18 @@ public function getComments($id)
             $this->client->api('issues')->show($this->getCurrentProject()->id, $id)
         );
 
-        return $issue->showComments();
+        $comments = [];
+        $issue->showComments();
```

Comment:
This looks a bit weird standing alone without using the return value. Is this intended?